app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (NotWearingMaskError,
+                        NotVaccinatedError,
+                        OutdatedVaccineError)
+
+
+def go_to_cafe(friends, cafe) -> str:
+    total = 0
+
+    for i in range(len(friends)):
```

Comment:
Didn't understand why you used range(len())... You can iterate directly over the list to use it's values. This is a better approach:
```suggestion
    for friend in friends:
```